src/main/java/svc/data/citations/CitationDAO.java

```diff
@@ -28,12 +28,12 @@
 		return Observable.merge(citationSearches).onExceptionResumeNext(Observable.just(null)).toList().toBlocking().first();
 	}
 
-	public List<Citation> getByLicenseAndDOB(String driversLicenseNumber, String driversLiscenseState, LocalDate dob) {
+	public List<Citation> getByLicenseAndDOB(String driversLicenseNumber, String driversLiscenseState, LocalDate dob, String lastName) {
```

Comment:
Please update the name of the method to reflect the params or make it more generic.
i.e. getByLicenseAndDOBAndLastName or, because all methods require DOB, just getByLicenseAndLastName